lib/Graph/Context.cpp

```diff
@@ -80,6 +80,15 @@ Placeholder *Context::getFirstUnallocated(std::list<Placeholder *> &lst) const {
   return nullptr;
 }
 
+uint64_t Context::dataSize() const {
+  size_t size = 0;
```

Comment:
`uint64_t` instead of `size_t`? Since that's the return type, and also type returned by `sizeInBytes()`.

Reply:
`getDataSize` as we also have `getSizeInBytes` and `getElementSize` in Glow's source code?